packages/react-vapor/src/components/button/examples/ButtonExamples.tsx

```diff
@@ -3,6 +3,7 @@ import {Svg} from '../../svg/Svg';
 import {Button} from '../Button';
 
 export class ButtonExamples extends React.Component<any, any> {
+    static description = 'Buttons are used to communicate and initialize actions when the user clicks them.';

```

Comment:
Suggestion:
Buttons communicate actions, and, when clicked, initialize those actions.